artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java

```diff
@@ -712,6 +712,19 @@ private synchronized void activate() throws Exception {
 
          serverLocator.start(server.getExecutorFactory().getExecutor());
       }
+      // add security role to disallow send/edit directly to sf queues
+      if (server.getConfiguration().isSecurityEnabled()) {
+         Set<Role> roles = new HashSet<>();
+         String addressMatch = storeAndForwardPrefix + name + "." + server.getConfiguration().getWildcardConfiguration().getSingleWordString();
```

Comment:
@gtully I'm not sure I understand. Can you be a bit more specific as to where should the rbac be set in?

Reply:
@gtully I was going to merge it now.. but it seems you still have some things to discuss on this one.


i will skip this one from the next apache artemis release I'm doing... it will be in the next one.